images/virtualization-artifact/pkg/controller/service/disk_service.go

```diff
@@ -148,6 +149,17 @@ func (s DiskService) CleanUpSupplements(ctx context.Context, sup *supplements.Ge
 		if err != nil && !k8serrors.IsNotFound(err) {
 			return false, err
 		}
+		pvc, err := s.GetPersistentVolumeClaim(ctx, sup)
```

Comment:
Check pvc == nil

Reply:
by the way, the DataVolume can be deleted by the hands of users. In this case, the ownerRef will still be in the in PVC